data-access/linked-mandataris.ts

```diff
@@ -150,7 +105,7 @@ export async function getDestinationGraphLinkedMandataris(
   if (result.results.bindings.length == 0) {
     return null;
   }
-  return result.results.bindings[0].dest.value;
+  return result.results.bindings[0].dest;
```

Comment:
in this case i would add typing to the function because right now it can't infer that this will be a term and I, as a naive user, would expect a string.